include/itkMorphologicalContourInterpolator.hxx

```diff
@@ -340,17 +340,17 @@ MorphologicalContourInterpolator< TImage >
     m_CrossDilator[threadId] = CrossDilateType::New();
     m_BallDilator[threadId] = BallDilateType::New();
     m_And[threadId] = AndFilterType::New();
-    m_And[threadId]->SetNumberOfThreads( 1 ); // excessive threading is counterproductive
+    m_And[threadId]->SetNumberOfWorkUnits( 1 ); // excessive threading is counterproductive
```

Comment:
We will get Python package build failures until we release ITK 5 RC 1, and push updated ITK source / build trees.